src/osx/cocoa/mediactrl.mm

```diff
@@ -495,7 +495,7 @@ - (AVPlayerLayer*) playerLayer
 
 bool wxAVMediaBackend::SetPosition(wxLongLong where)
 {
-    [m_player seekToTime:CMTimeMakeWithSeconds(where.GetValue() / 1000.0, 1)
+    [m_player seekToTime:CMTimeMakeWithSeconds(where.GetValue() / 1000.0, 60000)
```

Comment:
Thanks for the fix, but I'm almost as sure as I can be without actually testing it that this should be:
```suggestion
    [m_player seekToTime:CMTimeMakeWithSeconds(where.GetValue(), 1000)
```
because the actual time is `value / timescale`.

Could you please check if this really behaves as expected? TIA!

Reply:
It definitely works as I submitted it.  I'm pretty sure your change won't work as the first parameter to `CMTimeMakeWithSeconds()` is always seconds.

Reply:
Oh, sorry, I see my mistake now: I thought the arguments were `CMTime` `value` and `timescale` members, but actually they're `seconds` and (preferred) `timescale`. So yes, you're absolutely correct, sorry again.